src/components/DateTime/splDatetimeToIso.js

```diff
@@ -0,0 +1,46 @@
+export function splDatetimeToIso(datetime) {
+	// Check if datetime is a number or BigInt
+	if ((typeof datetime !== 'bigint') && (typeof datetime !== 'number')) {
+		return 'Invalid Date';
+	}
+	let BigIntDatetime = BigInt(datetime); // Convert to BigInt for correct bitwise operations
+
+	// Year extraction (bits 46-60)
+	let year = Number((BigIntDatetime >> BigInt(46)) & BigInt(0b111111111111)); // 14 bits
+	if (year >= 0b10_0000_0000_0000) {
+		year -= 0b10_0000_0000_0000; // Adjust for sign bit if necessary
+	}
+
+	// Month extraction (bits 42-46)
+	let month = Number((BigIntDatetime >> BigInt(42)) & BigInt(0b1111)); // 4 bits
+
+	// Day extraction (bits 37-42)
+	let day = Number((BigIntDatetime >> BigInt(37)) & BigInt(0b11111)); // 5 bits
```

Comment:
@aringocode why do you use `let` here and there? you dont change the value anywhere...